### ISSUE TITLE
Remove focus from browser search box when popup appears

### ISSUE THREAD

**Issue by seanblue** [FooSoft/yomichan#2081]
**Description**
When scanning for text in an HTML input, if text is found and the popup appears, the input is no longer focused even if it was previously. This is important because it allows single-key Yomichan shortcuts to work, when otherwise it would type into the input. Specifically, I reconfigured the "play audio" shortcut to be just Spacebar instead of one of the Alt key combo shortcuts.

However, when using the browser page search functionality (from Ctrl+F), scanning text on the page does *not* remove focus from the search box. This means that hitting Spacebar to play audio for the first Yomichan result in the popup does not work. Instead it just types into the search box. This isn't the end of the world, but I run into it a lot when searching for words in the books I'm reading.

I don't even know if it's possible for an extension to take focus away from the search box. If it is, please do so for consistency with how it works for HTML inputs and to fix the issue I'm having. If it's not possible, just let me know and close the issue.

**Browser version**
Chrome 98.0.4758

**Yomichan version**
22.2.2.0

**Comment by toasted-nutbread** [comment]
I don't believe there is any way to override the input focus of anything outside the webpage, so that would include the address bar, the search input, and other popup-based inputs that are controlled by the browser.
